### DISCUSSION TITLE
ElementTooltip usage

### DISCUSSION THREAD

**Discussion by kardoka** [iTwin/itwinjs-core#2354] [Q&A] [answered]
I want to add extra information in the built-in tooltips when you mouse-over a element. What are my options for doing so? Do I have to use the [ElementTooltip](https://www.itwinjs.org/reference/ui-framework/notification/elementtooltip/) component? If so, where in the code do I create the component so that it would override the default tooltips?

**Comment by bbastings** [comment, marked as answer]
Sample showing how to customize the element tooltip:
https://www.itwinjs.org/sample-showcase/?group=Viewer+Features&sample=tooltip-customize-sample&imodel=Retail+Building+Sample
If you are using the ui-framework you will have an implementation of the NotificationManager and just need to provide the tooltip contents.
